### ISSUE TITLE
Some movies dont loadup on Samsung Series 7

### ISSUE THREAD

**Issue by AdamRussak** [MediaBrowser/SamsungSmartTV#50]
some movies dont load UP, and looks like they load up and never running.
![image](https://user-images.githubusercontent.com/31593022/88665351-267d7380-d0e7-11ea-8145-f50282ec757d.png)

**Comment by LukePulverenti** [comment]
What version of the Samsung app?

**Comment by AdamRussak** [comment]
both the latest and the one before it.
I thought it was a problem in the latest version, so I reverted to the one before.
+ the same movie worked on mobile and web.

**Comment by LukePulverenti** [comment]
What version number exactly?

**Comment by AdamRussak** [comment]
currently installed: 1.0.78
Edit: 
TV Used: UE65RU7100UXSQ

**Comment by AdamRussak** [comment]
@LukePulverenti is there an update? do you need some more info from me?

**Comment by MarkBarbieri** [comment]
@AdamRussak, is it possible to get a copy of the file from your first post for testing?  It looks like it's got the audio track before the video track which may be the issue.

Has this always been an issue, or did it start recently?

**Comment by AdamRussak** [comment]
> @AdamRussak, is it possible to get a copy of the file from your first post for testing?  It looks like it's got the audio track before the video track which may be the issue.
> 
> Has this always been an issue, or did it start recently?

I didn't notice it before..
And what file would you like?

**Comment by MarkBarbieri** [comment]
This one please.

![](https://user-images.githubusercontent.com/31593022/88665351-267d7380-d0e7-11ea-8145-f50282ec757d.png)

**Comment by AdamRussak** [comment]
sorry for asking again, but do you want the movie file itself?

**Comment by MarkBarbieri** [comment]
> sorry for asking again, but do you want the movie file itself?

Yes please, a copy of the movie file if that’s possible, just for me to test

**Comment by AdamRussak** [comment]
> > @AdamRussak, is it possible to get a copy of the file from your first post for testing?  It looks like it's got the audio track before the video track which may be the issue.
> > Has this always been an issue, or did it start recently?
> 
> I didn't notice it before..
> And what file would you like?

here you go :)

https://www.transfernow.net/qrSt5H082020

**Comment by MarkBarbieri** [comment]
Ok, I've got that file now, thanks.

The first thing that I notice with that file is that the first track is the audio track, and not the video track.  This isn't entirely a problem, but it does make me suspicious as to how this file has been created/modified.

If I play that file on my 2017 Samsung TV, it appears not to play, however if I wait long enough (up to 2 minutes or longer) it does start playing with DirectPlay, but I have no audio.

I have tried swapping the tracks, but that does not resolve the issue.

If I wait for it to start playback, then drop the playback quality down to 720p - 5 Mbps to force it to transcode, the it plays correctly.  What is odd is that in this case is that whilst the video is being converted to the lower bitrate, the audio is actually being copied in its original format without conversion and it plays correctly.  However, I do notice that ffmpeg does report an error with the audio file when it is probed, so I expect it is rewriting the audio track header and correcting the error.

So in summary, there is a problem with this specific file:
- Tracks are not in their normal/correct order
- The audio track has a faulty header - I suspect this is what is causing the playback issues
- Forcing a transcode of the file resolve it

What I would suggest is that you try running this file through a tool such as Handbrake to convert and repair the audio file and  rewrite the movie file.  That should make it playable.

**Comment by LukePulverenti** [comment]
Are you still having an issue with this?

**Comment by AdamRussak** [comment]
After the comment by @MarkBarbieri  I removed the corrupt film and replaced it..

**Comment by LukePulverenti** [comment]
Thanks for the feedback.
